aioircd.py

```diff
@@ -285,7 +284,23 @@ async def JOIN(self, args):
 
     @command
     async def PRIVMSG(self, args):
-        pass
+        msg = args.split(":",1)
+        if msg[0] in [""," "]:
+            raise ErrNoRecipient("PRIVMSG")
+        if len(msg)<2 or msg[1] in [""," "]:
+            raise ErrNoTextToSend()
+        user,message = msg[2:]
+        if user.startswith("#"):
+            chann = self.local.channels.get(user)
+            if chann == None:
+                raise ErrNoSuchChannel(user)
+            await chann.send_all(f": {self.nick} PRIVMSG {user} :{message}")
```

Comment:
There should not be a space between the `:` and the nick

Reply:
👌